homeassistant/helpers/state.py

```diff
@@ -40,36 +20,11 @@
     STATE_UNKNOWN,
     STATE_UNLOCKED,
 )
-from homeassistant.core import Context, State, DOMAIN as HASS_DOMAIN
```

Comment:
We need to import the hass domain directly from core.py here:
https://github.com/home-assistant/home-assistant/blob/4863face6908fb4a942695aaa17feb58bad823e5/homeassistant/components/scene/__init__.py#L11